src/common/snippets/tests/src/lowered/pass/split_loops.cpp

```diff
@@ -0,0 +1,563 @@
+            n,
+            n_block,
+            std::vector<LoopPort>{LoopPort::create<PortType::NotProcessed>((*brgemm.first)->get_input_port(0)),
+                                  LoopPort::create<PortType::Incremented>((*brgemm.first)->get_input_port(1), 0)},
+            std::vector<LoopPort>{LoopPort::create<PortType::Incremented>((*brgemm.first)->get_output_port(0), 0)});
+        const auto blocking_n_loop_id = loop_manager->add_loop_info(blocking_n_loop);
+
+        (*add.first)->set_loop_ids({inner_add_loop_id});
+        (*brgemm.first)->set_loop_ids({blocking_n_loop_id});
+    }
+
+    {
+        auto param0 = linear_ir_ref->push_node<ov::op::v0::Parameter>(input_precision, input_shape_0);
+        auto param1 = linear_ir_ref->push_node<ov::op::v0::Parameter>(input_precision, input_shape_1);
+        auto param2 = linear_ir_ref->push_node<ov::op::v0::Parameter>(input_precision, input_shape_2);
+        auto brgemm = linear_ir_ref->push_node<ov::snippets::op::Brgemm>(param0.second, param1.second);
+        init_expr_descriptors(*brgemm.first, {brgemm_a_subtensor, brgemm_b_subtensor, brgemm_c_subtensor});
+        auto add = linear_ir_ref->push_node<ov::op::v1::Add>(brgemm.second, param2.second);
+        auto result = linear_ir_ref->push_node<ov::op::v0::Result>(add.second);
+
+        const auto& loop_manager = linear_ir_ref->get_loop_manager();
+        const auto brgemm_n_loop = std::make_shared<UnifiedLoopInfo>(
+            n,
+            n_block,
+            std::vector<LoopPort>{LoopPort::create<PortType::NotProcessed>((*brgemm.first)->get_input_port(0)),
+                                  LoopPort::create<PortType::Incremented>((*brgemm.first)->get_input_port(1), 0),
+                                  LoopPort::create<PortType::Incremented>((*add.first)->get_input_port(1), 0)},
+            std::vector<LoopPort>{LoopPort::create<PortType::Incremented>((*add.first)->get_output_port(0), 0)});
+        const auto blocking_n_loop_id = loop_manager->add_loop_info(brgemm_n_loop);
+
+        const auto add_n_split_loop = make_inner_split_loop_info(
+            n,
+            vector_size,
+            std::vector<LoopPort>{LoopPort::create<PortType::Incremented>((*add.first)->get_input_port(0), 0),
+                                  LoopPort::create<PortType::Incremented>((*add.first)->get_input_port(1), 0)},
+            std::vector<LoopPort>{LoopPort::create<PortType::Incremented>((*add.first)->get_output_port(0), 0)},
+            brgemm_n_loop);
+        const auto add_n_split_loop_id = loop_manager->add_loop_info(add_n_split_loop);
+
+        const std::map<size_t, size_t> loop_ids_mapper = {{blocking_n_loop_id, 1}, {add_n_split_loop_id, 3}};
+        const std::map<ExpressionPtr, std::vector<size_t>> expr_to_loop_ids = {
+            {*brgemm.first, {blocking_n_loop_id}},
+            {*add.first, {blocking_n_loop_id, add_n_split_loop_id}}};
+        assign_loop_ids(expr_to_loop_ids, loop_ids_mapper);
+    }
+}
+
```

Comment:
Can we test single iteration loops as well?

Reply:
Do you mean loops with `work_amount == increment`? Actually, in real cases we don't just insert such blocking loops (`BrgemmBlocking` pass just inserts full_dim in the corresponding subtensor values and don't create loops). And  taking into account that `SplitLoops` works mainly with blocking loops, it's worthless to add such test-cases: they don't cover real scenarios

Reply:
Got it